src/Mantasflowers.Services/ServiceAgents/Exceptions/OrderNotFoundException.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Mantasflowers.Services.ServiceAgents.Exceptions
+{
+    [System.Serializable]
+    public class OrderNotFoundException : System.Exception
+    {
+        public OrderNotFoundException() { }
```

Comment:
Why is this under _ServiceAgents_. Its not used within a service agent or related to it?